src/target_runtime.rs

```diff
@@ -0,0 +1,14 @@
+use std::sync::Arc;
+
+use async_graphql_value::ConstValue;
+
+use crate::{Cache, EnvIO, FileIO, HttpIO};
+
+#[derive(Clone)]
+pub struct TargetRuntime {
```

Comment:
Some doc about it's relevance would be very helpful!

Reply:
Added.